hunts/views.py

```diff
@@ -48,6 +49,11 @@ def stats(request, hunt_slug):
     solves_per_hour = hunt.get_solves_per_hour()
     minutes_per_solve = hunt.get_minutes_per_solve()
 
+    use_chart = can_use_chart(hunt)
+    chart_data = None
+    if use_chart:
```

Comment:
Nit: I don't think you need `use_chart` -- can just call `if can_use_chart(hunt):` directly IMO